x-pack/plugins/security_solution/cypress/integration/timeline_creation.spec.ts

```diff
@@ -47,14 +47,17 @@ import { openTimeline } from '../tasks/timelines';
 
 import { OVERVIEW_URL } from '../urls/navigation';
 
-// FLAKY: https://github.com/elastic/kibana/issues/79389
-describe.skip('Timelines', () => {
+describe('Timelines', () => {
   let timelineId: string;
 
-  before(() => {
+  beforeEach(() => {
     cleanKibana();
   });
 
+  after(() => {
+    if (timelineId) deleteTimeline(timelineId);
```

Comment:
Is this necessary? `cleanKibana` should delete this timeline on the next test execution, right?

Reply:
This step was already deleted :/

Reply:
You are right, I missed that one, thanks :)